infra/gateway/src/main/resources/application.yml

```
@@ -29,4 +29,8 @@ spring:
         - id: user-service
           uri: lb://user-service
           predicates:
-            - Path=/api/users/**
\ No newline at end of file
+            - Path=/api/users/**
+        - id: store-service
+          uri: lb://store-service
+          predicates:
+            - Path=/api/v1/stores/**, /api/v1/products/**, /api/v1/categories/**
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)